aws-postgresql.yml

```diff
@@ -62,14 +63,22 @@ provision:
     required: true
     type: number
     details: Size of storage volume for service instance.
-    default: 5
+    default: 100
```

Comment:
Changing this actually does not matter since the default is ignored when the field is required as this is

Reply:
I may leave the default here as 100 as you get an error attempting to create a plan with default storage type with less than 100GB storage. As you say, it's still required, but at least this gives a suggestion for a default value which will work, whereas if a user copies across a default of 5, and doesn't change storage_type the plan will fail.